index.js

```diff
@@ -0,0 +1,12 @@
+const humburger = document.querySelector('.humburger_menu');
+const navMenu = document.querySelector('.mobile_menu');
+
+humburger.addEventListener('click', () => {
+  humburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+//   console.log(humburger);
+});
```

Comment:
- [ ] Hi @Tafloninno, it would be nice if we don't have commented code in our codebase as that makes our code dirty and not professional. Kindly remove all commented unused code so your codebase looks clean and professional 👍🏽